package.json

```diff
@@ -40,12 +40,12 @@
     "suitcss-utils-display": "^0.4.0"
   },
   "scripts": {
-    "init": "npm run module && npm run virtualenv && npm run pip && npm install && npm run config",
+    "setup": "npm run module && npm run virtualenv && npm install && npm run config",
```

Comment:
This will need a corresponding update in CONTRIBUTING.md, since it currently tells you to run `[sudo] npm run init`.